charts/renku/values.yaml

```diff
@@ -542,7 +542,7 @@ notebooks:
     #   gitlab:
     #     clientId: jupyterhub
     #     clientSecret: dummy-secret
-    #     callbackUrl: '""' # Forces the default callback url
+    #     callbackUrl:  # Null value forces the default callback url
```

Comment:
Not sure you wanted this in this change in this commit/PR, but it's a useful fix!

Reply:
I think I'll leave it here - it causes many headaches.

Reply:
I agree, I have the same change in some half-baked chart PR too. Good to have it in!